e2e-tests/playwright/utils/UIhelper.ts

```diff
@@ -130,6 +145,15 @@ export class UIhelper {
     });
   }
 
+  async waitForH5Title(text: string) {
```

Comment:
Could we consider using uiHelper.verifyHeading() to validate the heading instead of creating a new one that waits for H5? I believe uiHelper.verifyHeading() will also wait for it and perform a visibility assertion as well.

Reply:
verifyHeading util is unable to identify the heading, which is why I had to create a separate util to test this

<img width="1230" alt="Screenshot 2024-02-21 at 12 08 19 AM" src="https://github.com/janus-idp/backstage-showcase/assets/22490998/5c8e6d62-8500-466b-9c8f-13adfbffcf6e">

Reply:
Perhaps increasing the wait time in `uiHelper.verifyHeading()`, such as `await expect(headingLocator).toBeVisible({timeout: });`, with the timeout as an optional parameter defaulting to 60 seconds, could help.

Reply:
using `verifyText` here